### PR TITLE
🌐 Add German translation for `docs/de/docs/tutorial/body-fields.md`

### PR DESCRIPTION
**Pull request by nilslindemann**
← `tutorial/body-multiple-params.md` (#10308)
→ `tutorial/body-nested-models.md` (#10313)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 05f33ba3eee51b9774a57263bcd8f8420d65d945 at: https://bc74ea94.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 55cd1bdbe4263dbe84e778034101df0f8209e583 at: https://28378b06.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 03d3dd2d90e7a597fb3e96175bd6e94511283f32 at: https://799f6f86.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit d990bbc36c84f8cbcc0f9f2195db7ed3c91e332e at: https://8c75477e.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5b05940dbeea739e4a102007d1374143149a70dc at: https://8b952cf5.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 92cfdf3d7e663fc850cf93f95b62573b0ec1a7cf at: https://4f69181f.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit aed065b0a2b5dd7a3c0160c7689523a1df65deda at: https://acfd432d.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 702eae5a984daf3d31fe9a884b2fb9a063ee8abb at: https://d81b3177.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit b5ae1d2ac553f0ff8e3f85f56dcf958b1cf11363 at: https://2d1bad45.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit c8d08b1ce8626f50db7ddf2b98cedf120f41a2d0 at: https://f93734cc.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 527216d57a5a8872ec063a8b6c7235a35eaf1aae at: https://5fd8d1f4.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6fb4def61071973327712eb807ca39b4e238af82 at: https://8da7b35d.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5898e12093f907e1f13da14a2757f23e89f4f4e5 at: https://55ed4dd9.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 990eda1fe278d00fad712bd86661de6954450000 at: https://ce1b045f.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit f120f33d3974bb578d1143621e175a7bada8801f at: https://db7f36ad.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit a5a24538c96af8a665a7bfe9b5ed01808182f9f7 at: https://c047878a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5d071c200b53d872076dfab4f771c54fc8d26e7f at: https://1a07c191.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8d4f748454e85d72161644aefd6122308f9fe49f at: https://211f2153.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit f1c4bb4a073b54e30f967d7c17dea1204bc359b2 at: https://5af3e5ab.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit c4648bc4a120ad7581509510164edac00c496d96 at: https://f748ae08.fastapitiangolo.pages.dev

File: docs/de/docs/tutorial/body-fields.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    Beachten Sie, dass jedes Modellattribut mit einem Typ, Defaultwert und `Field` die gleiche Struktur hat wie ein Parameter einer Pfadoperation-Funktion, nur mit `Field` statt `Path`, `Query`, `Body`.
+
+## Zusätzliche Information hinzufügen
+
+Sie können zusätzliche Information in `Field`, `Query`, `Body`, usw. deklarieren. Und es wird im generierten JSON-Schema untergebracht.
+
+Sie werden später mehr darüber lernen, wie man zusätzliche Information unterbringt, wenn Sie lernen, Beispiele zu deklarieren.
+
+!!! warning "Achtung"
+    Extra-Schlüssel, die `Field` überreicht werden, werden auch im resultierenden OpenAPI-Schema ihrer Anwendung gelistet. Da diese Schlüssel nicht notwendigerweise Teil der OpenAPI-Spezifikation sind, könnten einige OpenAPI-Tools, wie etwa [der OpenAPI-Validator](https://validator.swagger.io/), nicht mit Ihrem generierten Schema funktionieren.
```
Comment (mattkoehne) [inline docs/de/docs/tutorial/body-fields.md:109]: `ihrer Anwendung` -> `Ihrer Anwendung`

**Review by mattkoehne — CHANGES_REQUESTED** [review]
Thank you @nilslindemann 👍

**Comment by nilslindemann** [comment]
Auch hier, Danke fürs Review. :+1:

**Comment by tiangolo** [comment]
📝 Docs preview for commit cb0c8610b604e2ec7468dc9c1c2b71f7738609b9 at: https://b4086a49.fastapitiangolo.pages.dev

**Review by mattkoehne — APPROVED** [review]
Awesome! Thanks! 🥂

**Comment by alejsdev** [comment]
Thank you both for your help!  @nilslindemann @mattkoehne 🚀✨

**Pull request merged**